remotespark/utils/sparkevents.py

```diff
@@ -22,10 +22,10 @@ def emit_session_creation_start_event(self, session_guid, language):
         assert language in constants.SESSION_KINDS_SUPPORTED
 
         event_name = constants.SESSION_CREATION_START_EVENT
-        time_stamp = SparkEvents.get_utc_date_time()
+        time_stamp = "{}".format(SparkEvents.get_utc_date_time())
```

Comment:
1. `get_utc_date_time` should begin with `_` if it's private

2. Use `strftime` to format date strings